plugins/funind/gen_principle.ml

```diff
@@ -1524,7 +1524,7 @@ let derive_correctness (funs : Constr.pconstant list) (graphs : inductive list)
           let lemma =
             fst @@ Declare.Proof.by (Proofview.V82.tactic (proving_tac i)) lemma
           in
-          let () =
+          let (_ : GlobRef.t list) =
```

Comment:
```suggestion
          let (_ : _ list) =
```
(we only really want to ensure that the function is fully applied by using this type annot)

Reply:
Newer OCaml's warn about this anyways, but better safe than sorry.